File: Python PowerPoint Examples/SlidesApiDemo/moveSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "SlidesMove_1.pptx"
password = ""
storage = ""
slide_index = 1
new_position = 3
inputFolder = "input"
result =api.move_slide(fileName, slide_index=slide_index, new_position=new_position, password=password,storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/SlidesApiDemo/moveSlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "SlidesMove_1.pptx"; 
$slideIndex = 1;
$newPosition = 3;
$password = null; 
$folder = "Slides/SlidesMove/";
$storage = null; 
$apiInstance->moveSlide($name, $slideIndex, $newPosition, $password, $folder, $storage);